contributed_definitions/NXsts.nxdl.xml

```diff
@@ -552,31 +567,36 @@ should be NXcollection not NXprocess-->
 Time during which the spectroscopy data are acquired and averaged.-->
             <field name="integration_time" type="NX_NUMBER" units="NX_TIME">
                 <doc>
-                     Link to target:/NXentry/NXinstrument/NXenvironment/NXsweep_control/NXbias_spectroscopy/NXintegration_time
+                     Link to
```

Comment:
If there is splint, I suggest to do it after the word "target:", so a space shall be inserted

Reply:
An extra space after `target:` is added.